### PR TITLE
Add ssvm to wasm-engines benchmarks

### LINKED ISSUE #30: Support SSVM as a wasm engine for benchmarking
https://github.com/second-state/SSVM/releases

### PR DESCRIPTION
**Pull request by hugo-dc**
Closes #30.

### CODE REVIEW

File: benchmark_results_data/cpuinfo.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 model name	: Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
-cpu MHz		: 768.134
+cpu MHz		: 600.014
```
Comment (axic) [inline benchmark_results_data/cpuinfo.txt:2]: I like how the mhz goes down with every PR :wink:

**Comment by axic** [comment]
@hugo-dc @jwasinger @cdetrio why not merge this? (Well needs a rebase now, but still.)

**Comment by hugo-dc** [comment]
Rebased master

File: wasm-engines/Dockerfile [hunk 2]
```diff
@@ -3,8 +3,24 @@ FROM ubuntu:19.10
...
+
+RUN wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+RUN apt update && apt install -y \
+	libllvm9 \
+	llvm-9 \
+	llvm-9-dev \
+	llvm-9-runtime \
+	libclang-common-9-dev # for yaml-bench
+
+RUN git clone https://github.com/hugo-dc/SSVM.git --single-branch --branch bench
```
Comment (axic) [inline wasm-engines/Dockerfile:20]: Cannot this be used?

```suggestion
RUN git clone https://github.com/ewasm-benchmarking/SSVM.git --single-branch --branch bench
```

Comment (hugo-dc) [reply wasm-engines/Dockerfile:20]: I was planning to create a different PR updating the docker file to use the ewasm-benchmarking repositories, but you're right it makes sense to already make use of the ewasm-benchmarking/SSVM for.

Comment (axic) [reply wasm-engines/Dockerfile:20]: I'm fine with either way, its up to you.

Comment (hugo-dc) [reply wasm-engines/Dockerfile]: I updated the Dockerfile to use the ewasm-benchmarking fork.

**Comment by jwasinger** [comment]
Implemented in #43

**Pull request closed without merging**